eslzArm/managementGroupTemplates/policyDefinitions/policies.json

```diff
@@ -7113,7 +7143,7 @@
                         "mode": "All",
                         "description": "This policy denies the creation of a subnet with out a User Defined Route.",
```

Comment:
"This policy denies the creation of a subnet without a User Defined Route (UDR)."